scripts/EN/Cybil

```diff
@@ -1786,21 +1786,21 @@ DrawBackground(); // 0x23c8: 00000065
 PlaySoundFx(0x39, 0x1); // 0x23cc: 00002072
 ShowText((7080)"It was a slightly dirty, but familiar doll."); // 0x23d4: 0000100c
 HandleInput(); // 0x23da: 00000010
-ShowText((7081)"Countless blue fireworks went off in front of Cybil's eyes. Memories of the past... Unforgettable memories... Memories that should have been tightly sealed away within her..."); // 0x23de: 0000100c
+ShowText((7081)"Countless blue sparks danced in front of Cybil's eyes.¥n¥nW2Memories of the past…¥n¥nW2Unforgettable memories…¥n¥nW2Memories that should have been tightly sealed away within her…"); // 0x23de: 0000100c
 HandleInput(); // 0x23e4: 00000010
-ShowText((7082)"She was holding the doll tightly... Cybil had hidden in the closet... Through the crack, a familiar scene..."); // 0x23e8: 0000100c
+ShowText((7082)"She was holding the doll tightly…¥nW2Cybil had hidden in the closet…¥nW2Through the crack she saw..."); // 0x23e8: 0000100c
```

Comment:
Changed the end to "Through the crack she saw..." to match the correct Japanese.